tests/not_impl_gen.py

```diff
@@ -0,0 +1,33 @@
+objects = [
+    bool,
+    bytearray,
+    bytes,
+    complex,
+    dict,
+    float,
+    frozenset,
+    int,
+    iter,
+    list,
+    memoryview,
+    range,
+    set,
+    str,
+    tuple,
+    object
+]
+
+header = open("generator/not_impl_header.txt")
```

Comment:
Maybe a nice idea is to use ninja2 as template engine. This prevents seperate header, footer and body files. This introduces an extra dependency, but I think it is okay.